app/renderer/components/Inspector/GestureEditor.js

```diff
@@ -0,0 +1,565 @@
+import React, { Component } from 'react';
+import { Tabs, Input, Button, Card, Select, Row, Col, notification,
+         PageHeader, Space, Steps, Divider, Tooltip, Popover } from 'antd';
+import { PlayCircleOutlined, PlusCircleOutlined,
+         CloseOutlined, AimOutlined, RightCircleOutlined,
+         DownCircleOutlined, UpCircleOutlined, PauseCircleOutlined } from '@ant-design/icons';
+import { withTranslation } from '../../util';
+import { SCREENSHOT_INTERACTION_MODE, POINTER_TYPES } from './shared';
+import InspectorCSS from './Inspector.css';
+
+const {TabPane} = Tabs;
+const {Option} = Select;
+const {Step} = Steps;
+const ButtonGroup = Button.Group;
+const {POINTER_UP, POINTER_DOWN, PAUSE, POINTER_MOVE} = POINTER_TYPES;
+
+const COLORS = ['#FF3333', '#FF8F00', '#B65FF4', '#6CFF00', '#00FFDC'];
+const BUTTONS = {LEFT: 0, RIGHT: 1};
+const ACTION_TYPES = {ADD: 'add', REMOVE: 'remove'};
+const COORD_TYPE = {PERCENTAGES: 'percentages', PIXELS: 'pixels'};
+const TICK_PROPS = {POINTER_TYPE: 'pointerType', DURATION: 'duration', BUTTON: 'button', X: 'x', Y: 'y'};
+const CURSOR = {POINTER: 'pointer', TEXT: 'text'};
+const STATUS = {WAIT: 'wait', FINISH: 'finish', COLOR: '#FFFFFF', FILLER: 'filler'};
+const DISPLAY = {[POINTER_UP]: 'Pointer Up', [POINTER_DOWN]: 'Pointer Down',
+                 [PAUSE]: 'Pause', [POINTER_MOVE]: 'Move'};
+
+
+const DEFAULT_POINTERS = () => [{
+  name: 'pointer1',
+  ticks: [{id: '1.1'}],
+  color: '#FF3333',
+  id: '1',
+}];
+
+
+/**
+ * Shows the gesture editor interface
+ */
+class GestureEditor extends Component {
+
+  constructor (props) {
+    super(props);
+    this.interval = null;
+    this.state = {
+      name: this.props.loadedGesture ? this.props.loadedGesture.name : 'Untitled',
```

Comment:
We can't change it to that because its not guaranteed that there will be a loaded gesture because when we create a new gesture, `loadedGesture` is null and `loadedGesture.name` would throw an error

Reply:
in that case, `this.props.loadedGesture?.name || 'Untitled'`